deltachat-ios/Controller/PreviewController.swift

```diff
@@ -20,3 +23,53 @@ class PreviewController: QLPreviewControllerDataSource {
         return urls[index] as QLPreviewItem
     }
 }
+
+
+class BetterPreviewController: QLPreviewController {
```

Comment:
Can I suggest to rename the controller :)? Maybe MediaPreviewController or sth like that?